### ISSUE TITLE
command 'scratchpads.openScratchpad' not found

### ISSUE THREAD

**Issue by kankaristo** [buenon/scratchpads#2]
When I try to run any of the commands in this extension I get an error like:
`command 'scratchpads.openScratchpad' not found`

So, the extension can't be used at all.

**Comment by buenon** [comment]
Please attach the console output from dev tools

**Comment by kankaristo** [comment]
It's not finding `typesConfig.json`, but the file is actually called `typesconfig.json`. Linux and macOS are both case-sensitive, probably works on Windows?

```
console.ts:136 [Extension Host] Activating extension `buenon.scratchpads` failed:  ENOENT: no such file or directory, open '/home/sami/.vscode-insiders/extensions/buenon.scratchpads-0.0.3/typesConfig.json' (at /usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:446:325)
t.log @ console.ts:136
t._logExtensionHostMessage @ extensionHost.ts:393
(anonymous) @ extensionHost.ts:210
emitTwo @ events.js:106
emit @ events.js:194
process.nextTick @ internal/child_process.js:766
_combinedTickCallback @ internal/process/next_tick.js:73
_tickCallback @ internal/process/next_tick.js:104
console.ts:136 [Extension Host] Here is the error stack:  Error: ENOENT: no such file or directory, open '/home/sami/.vscode-insiders/extensions/buenon.scratchpads-0.0.3/typesConfig.json'
	at Object.fs.openSync (fs.js:584:18)
	at Object.module.(anonymous function) [as openSync] (ELECTRON_ASAR.js:173:20)
	at Object.fs.readFileSync (fs.js:491:33)
	at Object.fs.readFileSync (ELECTRON_ASAR.js:505:29)
	at loadFileTypes (/home/sami/.vscode-insiders/extensions/buenon.scratchpads-0.0.3/src/extension.js:125:39)
	at init (/home/sami/.vscode-insiders/extensions/buenon.scratchpads-0.0.3/src/extension.js:91:9)
	at activate (/home/sami/.vscode-insiders/extensions/buenon.scratchpads-0.0.3/src/extension.js:42:9)
	at Function.t._callActivateOptional (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:785:650)
	at Function.t._callActivate (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:785:329)
	at /usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:784:391
	at Object.g [as _notify] (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:71:572)
	at Object.enter (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:74:924)
	at n.Class.derive._oncancel._run (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:76:790)
	at n.Class.derive._oncancel._completed (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:76:231)
	at c (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:79:680)
	at K.then.o.(anonymous function) (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:79:894)
	at Object.g [as _notify] (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:71:572)
	at Object.enter (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:74:924)
	at n.Class.derive._oncancel._run (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:76:790)
	at n.Class.derive._oncancel._completed (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:76:231)
	at c (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:79:680)
	at K.then.o.(anonymous function) (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:79:894)
	at Object.g [as _notify] (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:71:572)
	at Object.enter (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:74:924)
	at n.Class.derive._oncancel._run (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:76:790)
	at n.Class.derive._oncancel._completed (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:76:231)
	at e.resolveOk (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:550:1000)
	at e._receiveReply (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:649:316)
	at e._receiveOneMessage (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:648:609)
	at /usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:647:452
	at /usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:650:575
	at /usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:90:599
	at e.fire (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:91:917)
	at u (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:156:14)
	at Socket.<anonymous> (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:156:191)
	at emitOne (events.js:96:13)
	at Socket.emit (events.js:191:7)
	at readableAddChunk (_stream_readable.js:178:18)
	at Socket.Readable.push (_stream_readable.js:136:10)
	at Pipe.onread (net.js:560:20) (at /usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:446:393)
```

**Comment by kankaristo** [comment]
If I rename the file to `typesConfig.json`, the extension seems to work correctly.

**Comment by buenon** [comment]
Fixed in version 0.0.5
